src/config.ts

```diff
@@ -80,13 +82,22 @@ export interface ResolvedConfig {
   AddressType: IsUnknown<Config['AddressType']> extends true
     ? DefaultConfig['AddressType']
     : Config['AddressType']
+
+  /**
+   * TypeScript type to use for `bytes` values
+   * @default `0x${string}`
+   */
+  BytesInputType: IsUnknown<Config['BytesInputType']> extends true
```

Comment:
Another idea for the config type is keeping the name `BytesType`, but allowing it to be an object:

```ts
export interface DefaultConfig {
  BytesType: {
    Inputs: `0x${string}` | Uint8Array
    Outputs: `0x${string}`
  }
}
```

Reply:
I like this approach better. I will go with this. Will need to remove the capital lettering of Inputs and Outputs in order to keep with the abi spec as well as using `AbiParameterType` as keys for the object. This approach also works best with the `TypedData` types. 

Will also update the read me to reflect this change.